pdns/dnsdistdist/docs/eol.rst

```diff
@@ -0,0 +1,38 @@
+End of life statements
+======================
+
+.. list-table:: PowerDNS dnsdist Release Life Cycle
+   :header-rows: 1
+
+   * - Version
+     - Release date
+     - Security-Only updates
+     - End of Life
+   * - 1.6
+     - May 11 2021
+     - 
+     - 
+   * - 1.5
+     - July 30 2020
+     - 
+     - 
+   * - 1.4
+     - November 20 2019
+     - ~ April 2021
+     - 
+   * - 1.3
+     - March 30 2018
+     - 
+     - ~ April 2021
```

Comment:
Both are EOL per May 2021, and 1.5 will be ' critical sec only ' per approximately november 2021